scene/gui/text_edit.cpp

```diff
@@ -4226,7 +4226,10 @@ double TextEdit::get_scroll_pos_for_line(int p_line, int p_wrap_index) const {
 		return p_line;
 	}
 
-	double new_line_scroll_pos = get_visible_line_count_in_range(0, CLAMP(p_line, 0, text.size() - 1));
+	double new_line_scroll_pos = 0.0;
+	if (p_line > 0) {
+		new_line_scroll_pos = get_visible_line_count_in_range(0, CLAMP(p_line - 1, 0, text.size() - 1));
+	}
```

Comment:
This makes the `CLAMP` redundant since `p_line - 1` can no longer be less than 0, so it should be `MIN(p_line - 1, text.size())`.

Reply:
Fixed :+1: